tests/helpers.py

```diff
@@ -2652,3 +2652,18 @@ def default_volumesnapshotclass(interface_type):
         resource_name=resource_name
     )
     return OCS(**base_snapshot_class.data)
+
+
+def get_snapshot_content_obj(snap_obj):
```

Comment:
better to avoid adding more functions to tests/helpers.py
Please move to a relevant location under ocs_ci/ and also, docstring is missing

Reply:
Done